scm-webapp/src/main/java/sonia/scm/api/v2/resources/IndexDto.java

```diff
@@ -34,8 +37,16 @@
 
   private final String version;
 
+  @JsonInclude(JsonInclude.Include.NON_EMPTY)
+  private final String initialization;
+
   IndexDto(Links links, Embedded embedded, String version) {
+    this(links, embedded, version, null);
+  }
```

Comment:
Is this deprecated?

Reply:
No, I wouldn.t say so, this is still legit.